### PR TITLE
 Fix #248: add `maybe_update_issue_points` step 

### LINKED ISSUE #248: Sync story points from bugzilla to Jira
As per summary. Both JIRA and bugzilla have points values. It would be nice if changes in bugzilla were reflected in JIRA.

### PR DESCRIPTION
**Pull request by leplatrem**
* Requires #247 

```python
>>> [f for f in s.client.get_all_fields() if "points" in f["name"].lower()]
[
{'id': 'customfield_10678', 'key': 'customfield_10678', 'name': 'Points Remaining', 'untranslatedName': 'Points Remaining', 'custom': True, 'orderable': True, 'navigable': True, 'searchable': True, 'clauseNames': ['cf[10678]', 'Points Remaining', 'Points Remaining[Number]'], 'schema': {'type': 'number', 'custom': 'com.atlassian.jira.plugin.system.customfieldtypes:float', 'customId': 10678}}, 

{'id': 'customfield_10310', 'key': 'customfield_10310', 'name': 'Original Story Points', 'untranslatedName': 'Original Story Points', 'custom': True, 'orderable': True, 'navigable': True, 'searchable': True, 'clauseNames': ['cf[10310]', 'Original Story Points', 'Original Story Points[Number]'], 'schema': {'type': 'number', 'custom': 'com.atlassian.jira.plugin.system.customfieldtypes:float', 'customId': 10310}}, 

{'id': 'customfield_10037', 'key': 'customfield_10037', 'name': 'Story Points', 'untranslatedName': 'Story Points', 'custom': True, 'orderable': True, 'navigable': True, 'searchable': True, 'clauseNames': ['cf[10037]', 'Story Points', 'Story Points[Number]'], 'schema': {'type': 'number', 'custom': 'com.atlassian.jira.plugin.system.customfieldtypes:float', 'customId': 10037}}]

```

```python 
>>> s.client.update_issue_field(key="FIDEFE-4686", fields={"customfield_10037": 1})
>>>
```

Screenshot 2024-03-05 at 18 28 24

### CODE REVIEW

File: jbi/models.py [hunk 1]
```diff
@@ -74,9 +74,27 @@ class ActionParams(BaseModel, frozen=True):
...
+        "?": 0,
+        "1": 1,
+        "2": 2,
+        "3": 3,
+        "5": 5,
+        "7": 7,
+        "8": 8,
+        "12": 12,
+        "13": 13,
+        "15": 15,
```
Comment (alexcottner) [inline jbi/models.py:96]: I think I'm contractually obligated to call out that 7, 12, and 15 are not valid story point amounts.

File: tests/unit/test_steps.py [hunk 2]
```diff
@@ -728,6 +728,105 @@ def test_change_to_unknown_resolution_with_resolution_map(
...
+    )
+
+    mocked_jira.create_issue.assert_not_called()
+    mocked_jira.update_issue_field.assert_called_with(
+        key="JBI-234", fields={"customfield_10716": {"value": "Moderate"}}
+    )
+
+
+def test_update_issue_points(
+    action_context_factory,
```
Comment (alexcottner) [inline tests/unit/test_steps.py:800]: Would like to see a test where we provide an unmapped value for story points just to keep behavior consistent going forward.

**Review by alexcottner — APPROVED** [review]
Would like to see one more test added but looks good overall

File: jbi/models.py (hunk 1 above)
Comment (leplatrem) [reply jbi/models.py:96]: I just copied the dropdown values from Bugzilla UI:

Screenshot 2024-03-06 at 10 25 28

**Pull request merged**